java/client/src/org/openqa/selenium/remote/mobile/RemoteNetworkConnection.java

```diff
@@ -42,7 +42,7 @@ public ConnectionType getNetworkConnection() {
   @Override
   public ConnectionType setNetworkConnection(
       ConnectionType type) {
-    Map<String, ConnectionType> mode = ImmutableMap.of("type", type);
+    Map<String, Integer> mode = ImmutableMap.of("type", type.getBitMask());
```

Comment:
Reflection!? Oh well, when in Rome... Have removed getBitMask, and added toJson method. Thanks.